problems.h

```diff
@@ -8,27 +9,172 @@ using namespace std;
 int reverseInt(int num)
 {
     // Your code here
+    bool Is_Negative = 0;
+    if(num<0){
+        Is_Negative = 1;
+        num = abs(num);
+    }
+    string st1 = to_string(num);
+
+      Stack s1 = Stack(st1[0]);
+   for(int i=1;i<st1.length();i++){
+      s1.push(st1[i]);
+   }
+    string answer;
+    while(!s1.isEmpty()){
+        answer += s1.pop();
+    }
+    int a = stoi(answer);
+     if(Is_Negative){
+       a*=-1;
+    }
+    return a;
+
+    
+
 }
 
 // Returns -1 if celebrity
 // is not present. If present,
 // returns id (value from 0 to n-1).
-int findCelebrity(int n, vector<vector<int> >& matrix)
+int findCelebrity(int n, vector<vector<int>>& matrix)
 {   
     // Your code here
+    bool not_celebrity = 0;
+
+    Stack<int> s2 = Stack(0);
+    for(int i=1;i<n;i++){
+        s2.push(i);
+    }
+    
+    while(s2.top->down!=nullptr){
+        int a = s2.pop();
+        int b = s2.pop();
+        if(matrix[a][b]==1){
+            s2.push(b);
+        }else{
+            s2.push(a);
+        }
+    }
+
+    int help=s2.peek();
+
+    for(int i=0;i<n;i++){
+        if(help==i){
```

Comment:
people shouldn't know themselves so I don't know that this matters, but still nice work!